From d5fd900c3b3191c0842532633d084bb968bc13bf Mon Sep 17 00:00:00 2001
From: Mike Lothian <mike@fireburn.co.uk>
Date: Mon, 8 Jun 2020 02:43:42 +0100
Subject: [PATCH] Add limits to gamepad

---
 device/gamepad/public/cpp/gamepad.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/device/gamepad/public/cpp/gamepad.h b/device/gamepad/public/cpp/gamepad.h
index 67e793a23746..c2ead1bd31a7 100644
--- a/device/gamepad/public/cpp/gamepad.h
+++ b/device/gamepad/public/cpp/gamepad.h
@@ -7,6 +7,7 @@
 
 #include <stddef.h>
 #include <cstdint>
+#include <limits>
 
 #include "base/component_export.h"
 #include "base/strings/string16.h"
-- 
2.27.0

